reference/segments.v3.yml

```diff
@@ -330,7 +330,10 @@ paths:
       summary: 'Get All Shopper Profiles'
       tags:
         - Shopper Profiles
-      description: Returns a paginated *Shopper Profiles* list.
+      description: |
+          Returns a paginated *Shopper Profiles* list. 
+
+          **NOTE**: The `modify` Customers OAuth scope is a requirement for this endpoint.

```

Comment:
```suggestion
```
I am not sure if this is useful now that we added the same to the Auth section

Reply:
DEVDOCS would like to keep this callout in case developers don't look at the Auth details.